.github/workflows/.rubocop.yml

```diff
@@ -0,0 +1,32 @@
+# The behavior of RuboCop can be controlled via the .rubocop.yml
+# configuration file. It makes it possible to enable/disable
+# certain cops (checks) and to alter their behavior if they accept
+# any parameters. The file can be placed either in your home
+# directory or in some project directory.
+#
+# RuboCop will start looking for the configuration file in the directory
+# where the inspected file is and continue its way up to the root directory.
+#
+# See https://docs.rubocop.org/rubocop/configuration
+
+Enabled: true
+
+name: RuboCop
+on: [pull_request]
```

Comment:
Can you also add `push` here? And also rename the file so it's not a hidden file? Not sure if it will work with the dot.

Reply:
Sure

Reply:
Fixed 👍🏼